README.md

```diff
@@ -109,8 +109,34 @@ export ETH_SIGNING_KEY=<Your ETH SIGNING KEY>
 pigeon start
 ```
 
-- Remember to run pigeon as a systemd Service! If you have a good systemd service implementation for Ubuntu, please make a PR on this README and we will add it.
-
+#### Using systemd service
+Create service file
+
+```shell script
+tee <<EOF >/dev/null /etc/systemd/system/pigeond.service
```

Comment:
@aditya-manit this should be `pigeon`, not `pigeond`

Reply:
Usually I have seen a `d` suffix in binaries and services too, `d` for daemon I guess. We can remove it if we want to - nothing major.

Reply:
the prefix `d` is a cosmos thing where they add the `d` at the end of a chain executable